### PR TITLE
Add pipeline for adar-autotest

### PR DESCRIPTION
**Pull request by C4tWithShell**
# Task

[DOPS-2438]: Pipeline for adar-autotest.

## Changes

1. Add pipeline, that will be triggered by deploy of adar-test app.

[Pipeline](https://jenkins.soramitsu.co.jp/job/adar/job/qa/job/deploy/job/autotests/)

## Author

Signed-off-by: Vladimir Shelkovnikov <shelkovnikov@soramitsu.co.jp>

[DOPS-2438]: https://soramitsu.atlassian.net/browse/DOPS-2438

### CODE REVIEW

**Review by RamilMus — DISMISSED** [review]

**Comment by f33r0** [comment]
DEBIAN_FRONTEND=noninteractive will help you

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        stage('Run tests') {
+            steps {
+                script {
+                    docker.withRegistry( registryUrl, registryCredentialsId ){
+                        docker.image( "docker.soramitsu.co.jp/build-tools/" + agentImage ).inside(){
+                            sh 'apt update'
+                            sh 'npm i'
+                            sh 'npx playwright install --with-deps'
+                            sh 'npx playwright test'
+
```
Comment (Cre-eD) [inline Jenkinsfile]:
```suggestion
                            sh '''
                            npm i
                            npx playwright install --with-deps
                            npx playwright test
                            '''
```

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    stages {
+        stage('Run tests') {
+            steps {
+                script {
+                    docker.withRegistry( registryUrl, registryCredentialsId ){
+                        docker.image( "docker.soramitsu.co.jp/build-tools/" + agentImage ).inside(){
+                            sh 'npm i'
+                            sh 'npx playwright install --with-deps'
+                            sh 'npx playwright test'
+
```
Comment (Cre-eD) [inline Jenkinsfile:30]:
```suggestion
                            sh '''
                              npm i
                              npx playwright install --with-deps
                              npx playwright test
                            '''
```

**Review by Cre-eD — APPROVED** [review]
Good job!

**Pull request merged**